EdFi.FIF.API/test/EdFi.FIF.Data.Tests/EdFi.FIF.Data.Tests.csproj

```
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
```

In this case I agree.

```suggestion
    <IsPackable>false</IsPackable>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    <Copyright>(c) 2020, Ed-Fi Alliance, LLC, and contributors</Copyright>
```